.gitignore

```diff
@@ -0,0 +1 @@
+local/
```

Comment:
Is this a debug line? Does it need to be committed?

Reply:
We can remove it. I found it convenient to have local testing directory which isn't checked into git.